configure.ac

```diff
@@ -75,6 +75,12 @@ AC_ARG_WITH(stage1-default-location,
             [RKT_STAGE1_DEFAULT_LOCATION="${withval}"],
             [RKT_STAGE1_DEFAULT_LOCATION=auto])
 
+AC_ARG_WITH(stage1-default-images-directory,
+            [AS_HELP_STRING([--with-stage1-default-images-directory],
+                            [default directory for stage1 images, default: LIBDIR/rkt/stage1-images])],
```

Comment:
Ok. As long as it is explained in the "Note for packagers" section of the release notes.